arch/arm/core/aarch64/mmu/arm_mmu.c

```diff
@@ -436,7 +439,7 @@ SYS_INIT(arm_mmu_init, PRE_KERNEL_1,
 int arch_mem_map(void *virt, uintptr_t phys, size_t size, uint32_t flags)
```

Comment:
need to rebase, arch_mem_map() is now void. the arch is expected to either pre-allocate page tables for the entire address space defined by CONFIG_KERNEL_VM_BASE/CONFIG_KERNEL_VM_SIZE (so that mappings cannot ever fail) or panic/assert in the arch code if it runs out of memory for paging structures